### ISSUE TITLE
Upon first episode posting, hitting a 404, despite seeing the preview post (before posting) without problems.

### ISSUE THREAD

**Issue by accozzaglia** [podlove/podlove-publisher#695]
I'll start by noting that the category taxonomy is correct — both from the WP side and from the PL side: in both, there is agreement that the location for episode posts happens at http://accozzaglia.ca/dj/podcasts/.

Unfortunately, after posting publicly, the post appears on the main blog list as the latest blog post on my general blog. Upon clicking to view the post, I get a 404.

The PL expert settings are configured as follows::

`[yes] Combine blog & podcast: Include episode posts on the front page and in the blog feed`
`[no] Hide blog feeds: Hide default WordPress feeds for blog and comments (no auto-discovery).`
`[yes] Permalink structure for episodes: Use the same permalink structure as posts`
`[yes] Episode pages: Enable episode pages: a complete, paginated list of episodes, sorted by publishing date.`

`(path:) [http://accozzaglia.ca]/dj/podcasts/`

On another github PL thread closed some time ago, another user was running into trouble with their theme. I recall with the theme I use of having to add a new post category for another module of my site (woocommerce, and adding the post category 'product' to functions.php resolved that issue, although it mostly related to thumbnail resizing parameters). There is no analogue with my theme that would be similar to the issue mentioned above.

tl;dr:

+ slug for the podcast is '2355'
+ slug for this post is '2355001'
+ prior to publishing the post, I can *preview* the post as it would appear after it would be posted. Everything looks great there.
+ The permalink, according to the New/Edit Episode page, is http://accozzaglia.ca/dj/podcasts/2355001/ (which appears to be correct)
+ The Category taxonomy for WP's Categories features the hierarchy of 'dj' —> 'podcasts'
+ Posting location on PL reads as `[http://accozzaglia]/dj/podcasts`
+ Browsing to the http://accozzaglia.ca/dj/podcasts/ category page shows the posting title as an existing post, as it also does on the main blog.
+ Clicking on that post link, however, produces a 404.
+ unlike previewing before posting, previewing *after* publishing produces a 404
+ W3TC caching plugin was disabled after clearing all local caches; I also performed a full cache purge at my CDN; lastly, I tried the PL repair button. 
+ No improvement on all three counts.

What would you need from me to look into this further?

EDIT: I'm leaving the post "live" for now, even though it's throwing off visitors to my site who wonder what the 404 is all about.

EDIT #2: Adding the following for reference:

`Website             http://accozzaglia.ca`
`PHP Version         5.4.34`
`WordPress Version   3.6.1`
`Publisher Version   2.0.5`
`Web Player Version  2.0.17`
`curl Version        7.24.0`
`iconv               available`
`simplexml           ok`
`max_execution_time  300`
`upload_max_filesize 50M`
`memory_limit        256M`
`disable_classes     `
`disable_functions   `
`permalinks          ok`
`podcast_settings    `
`web_player          ok`
`podlove_cache       on`
`assets              `
`	m4a    audio/mp4`
`	jpg    image/jpeg`

`0 errors`
`0 notices`
`Nice, Everything looks fine!`

EDIT: #3: the theme in use is a variation of Pinboard 1.06. Also tried disabling all plug-ins, save for PL, flushed the cache, and tried to reload the 404'd post, but this was ineffective.

EDIT: #4: tried a new approach of disabling one plug-in — PL — then deleting the 'podcasts' taxonomy category. I did this in case there was some kind of collision between WP's handling of taxonomy terms and the way PL is trying to access the same term as possibly a built-in term. When I reactivated PL without an existing 'podcasts' category, I tried to post the episode under the 'Uncategorized' category. When I did this, the post finally appears. So I tried assigning a different random category for the episode; as with "Uncategorized", the post does appear successfully. So then I tried to create a new category 'podcasts' to the categories list from within the episode post page (exactly where the previous 'podcasts' category had been before I deleted it (as a child of category 'dj'). Then I tried posting again. 404.

Is there somewhere within the PL code which insists that it have domain over the term 'podcasts' so that a WP admin cannot use 'podcasts' as its own category? This seems to be my guess. Whatever the case, I want to resolve this, because having my standard path read as '[domain]/dj/podcasting' is better for SEO and semantic recall. Thanks.

EDIT #6: (seriously, you guys. I'm trying to be as thorough with QA here as I can mostly helpfully be). So upon continued testing of placing the episode post into different categories, the post appears successfully, but so long *IF* the category isn't nested under another category (e.g., `[site URL]/foo/[podcast episode]` is OK, but `[site URL]/foo/bar/[podcast episode]` is NOT OK). Why does PL get derailed by this?

**Comment by eteubert** [comment]
I tried to reproduce your issue but for me everything works fine.
Let me list my settings, maybe you see what I did differently from you:

Expert Settings

![screen shot 2015-03-12 at 13 35 38](https://cloud.githubusercontent.com/assets/235918/6612732/b24e792e-c8bc-11e4-8308-1441c870e427.png)

Permalinks

`/%category%/%postname%/`

Episode Categories (via the Podlove module, *not* WordPress categories)

![screen shot 2015-03-12 at 13 36 47](https://cloud.githubusercontent.com/assets/235918/6612744/eb142380-c8bc-11e4-919e-0d00dda6f5fe.png)

In an example episode, I chose the "podcast" category. Now:

* all episodes are listed under `/dj/podcasts/`
* the example episode's permalink is `/dj/podcasts/newz085-cuba-invented-the-podcast/`
